### ISSUE TITLE
Boolean type checking is missing

### ISSUE THREAD

**Issue by tghw** [padolsey-archive/jquery.lint--old#2]
Code:
    $(...).toggleClass('hidden', jAvailableAreas.length == 0);

Warning:
    More info:
    You passed: ["hidden", false]
    Available signatures include:
    toggleClass(className)
    toggleClass(className, switch)

We're passing a string and a boolean, which is one of the options. Looks right to me. jQuery 1.3.2.

**Comment by tghw** [comment]
I'm seeing a similar issue with outerWidth, which takes an optional boolean. We're passing `true`. Could the boolean type checking have a bug?

**Comment by tghw** [comment]
After digging into the code a bit, I'm pretty sure the problem is that Boolean is not in the type map. Adding the following removed the spurious warnings:

```
diff --git a/jquery.lint.js b/jquery.lint.js
--- a/jquery.lint.js
+++ b/jquery.lint.js
@@ -215,7 +215,10 @@
                 return isFunction(o);
             },
             String: function(o) {
                 return typeof o === 'string';
+            },
+            Boolean: function(o) {
+                return typeof o == 'boolean';
             },
             Number: function(o) {
                 return typeof o === 'number' && !isNaN(o);
```

**Comment by padolsey** [comment]
Added type boolean. Thanks for reporting this!
